specs/latest/2.0/index.html

```diff
@@ -699,12 +700,18 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum MAX_CLIENT_WAIT_TIMEOUT_WEBGL                 = 0x9247;
 
   /* Buffer objects */
+  void bufferData(GLenum target, ByteBuffer? data, GLenum usage,
+                  optional unsigned long byteOffset = 0);
+  void bufferSubData(GLenum target, GLintptr offset, ByteBuffer data,
+                     optional unsigned long byteOffset = 0);
+
   void copyBufferSubData(GLenum readTarget, GLenum writeTarget, GLintptr readOffset,
                          GLintptr writeOffset, GLsizeiptr size);
   // MapBufferRange, in particular its read-only and write-only modes,
   // can not be exposed safely to JavaScript. GetBufferSubData
   // replaces it for the purpose of fetching data back from the GPU.
-  void getBufferSubData(GLenum target, GLintptr offset, ArrayBuffer? returnedData);
+  void getBufferSubData(GLenum target, GLintptr offset, ByteBuffer returnedData,
+                        optional unsigned long byteOffset = 0);
```

Comment:
All of the other WebGL 2.0 entry points produce OpenGL-style errors. There is no reason for this one to be an exception. (The asynchronous version proposed in #1937 has to produce DOM-style errors, so it's been changed to not produce OpenGL-style errors.)